samples/resourceops/resourceop_basic.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from kubernetes import client as k8s_client
+import kfp.dsl as dsl
+
+
+@dsl.pipeline(
+    name="ResourceOp Basic",
+    description="A Basic Example on ResourceOp Usage."
+)
+def resourceop_basic(username, password):
```

Comment:
Could you also add the samples to e2e tests at https://github.com/kubeflow/pipelines/blob/master/test/e2e_test_gke_v2.yaml#L119?

Reply:
OK, we'll do this gladly.
Can we not block this PR for this, as it has been pending for quite some time?

We will expand all e2e testing to work with volumes, but to do this, we will also have to update the test environment so it uses an updated Argo image. We need the patch for argoproj/argo#1238, so we either use a custom-built image, or wait for the release of v2.3.

We'll open a new issue and discuss this right after this PR.

Reply:
OK, we will follow up with a PR extending the way E2E tests run, using the newer Argo image, and we will discuss all the details there, right after this one has been merged.